src/detect-engine-analyzer.c

```diff
@@ -841,6 +842,68 @@ void EngineAnalysisRules2(const DetectEngineCtx *de_ctx, const Signature *s)
     }
     jb_close(ctx.js);
 
+    jb_open_object(ctx.js, "policy");
```

Comment:
@jasonish @jufajardini we should probably consider this and the verdict output discussion we had. It would be good to make these things similar if not the same, and at least use one set of keywords.

Reply:
Given we brainstormed on this for a while, can you fit this new "policy" into https://github.com/OISF/suricata/pull/8596#discussion_r1197787688?

Reply:
I'm sorry, this escaped me completely last week. 
I noticed that this part seems to not have made it to the merged PR (https://github.com/OISF/suricata/pull/8928).

Well defining the keywords is one of the first steps, in my opinion, as at the moment 'policy', for instance, only echoes "exception policy" in my head xD

So in engine analysis mode, one would be able to see all actions that are available for rules, and what each of them means, if triggered (aka the 'verdict')?